### PR TITLE
Foreman 3.4 build updates

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

File: rel-eng/tito.props [hunk 1]
```diff
@@ -546,9 +546,9 @@ whitelist =
   tracer
   yggdrasil
 
-[foreman-client-nightly-rhel7]
+[foreman-client-3.4-rhel7]
 disttag = .el7
-# don't whitelist foreman-release here, it's already in foreman-nightly-nonscl-rhel7
+# don't whitelist foreman-release here, it's already in foreman-3.4-nonscl-rhel7
```
Comment (evgeni) [inline rel-eng/tito.props:551]: now that you say it, there is no 3.4-nonsc-7, so you NEED to whitelist it here!

same should happen for nightly, I geuss

File: repoclosure/yum.conf [hunk 2]
```diff
@@ -153,6 +153,6 @@ failovermethod=priority
 name=f29-puppet-6
 baseurl=http://yum.puppetlabs.com/puppet6/fedora/29/x86_64/
 
-[f29-foreman-client-nightly]
-name=Foreman Client nightly F29
-baseurl=http://koji.katello.org/releases/yum/foreman-client-nightly/fc29/x86_64/
+[f29-foreman-client-3.4]
+name=Foreman Client 3.4 F29
+baseurl=http://koji.katello.org/releases/yum/foreman-client-3.4/fc29/x86_64/
```
Comment (evgeni) [inline repoclosure/yum.conf:158]: there is no f29, just zuul

**Review by evgeni — CHANGES_REQUESTED** [review]
doop doop doop

File: repoclosure/yum.conf (hunk 2 above)
Comment (pcreech) [reply repoclosure/yum.conf]: f29 is gone

File: rel-eng/tito.props (hunk 1 above)
Comment (pcreech) [reply rel-eng/tito.props]: done

**Pull request merged**